doc/read-the-docs-site/RELEASE.adoc

```diff
@@ -71,8 +71,10 @@ This updates versions and version bounds, and assembles the changelogs.
 3. Tag `x.y.z.0-rc1` on master.
 - There's no need to create a release branch at this point.
 A release branch is only needed if bugs are found in `x.y.z.0-rc1` and we need to backport a fix from master.
-4. Run the release QA process.
-5. If no release blocking issue is found, tag `x.y.z.0` and upload the packages to https://github.com/input-output-hk/cardano-haskell-packages[CHaP].
+- This step can be omitted if no additional tests and checks are needed in the release QA process. See "Release QA Process" below.
+4. Run the release QA process. See "Release QA Process" below.
+5. If no release blocking issue is found, open a PR for updating the cabal files and the changelog files.
```

Comment:
I'm confused, I think it's implied that this is done by step 1? Or at least it seems to be elsewhere. Perhaps we need to clarify when things go into master, reading the list it's unclear. e.g. step 3 says to tag on master, but does that include the release preparation steps or not?